keyboards/bananasplit/bananasplit.h

```diff
@@ -30,21 +43,30 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D },       \
     { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, KC_NO },     \
     { K30, KC_NO,    K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D },  \
-    { K40, K41, K42, KC_NO,    K44, K45, K46, KC_NO,    K48, K49, K4A, K4B, K4C }   \
+    { K40, K41, K42, KC_NO,    K44, K45, K46, KC_NO,    K48, K49, K4A, K4B, K4C, KC_NO }   \
```

Comment:
The original macro doesn't have this extra addition on the end. What does this do?

https://github.com/evangs/tmk_keyboard/blob/master/keyboard/bananasplit/keymap_common.h#L40